### PR TITLE
Micrometer gauge number

### PR DESCRIPTION
**Pull request by Channyboy**
Related to https://github.com/eclipse/microprofile-metrics/pull/699
Depends on :

FIRST: https://github.com/smallrye/smallrye-metrics/pull/513
(Current PR contains the commits of above )

Relevant commit is (or whatever the last commit is ):  https://github.com/smallrye/smallrye-metrics/pull/517/commits/aa0a78dfbca206a4225f0d215f773b2a3a3d1895

### CODE REVIEW

File: implementation/src/main/java/io/smallrye/metrics/exporters/OpenMetricsExporter.java [hunk 1]
```diff
@@ -37,17 +35,19 @@ public String exportAllScopes() {
         for (MeterRegistry meterRegistry : prometheusRegistryList) {
             PrometheusMeterRegistry promMeterRegistry = (PrometheusMeterRegistry) meterRegistry;
             //strip "# EOF"
-            sb.append(promMeterRegistry.scrape(TextFormat.CONTENT_TYPE_OPENMETRICS_100).replaceFirst("\r?\n?# EOF", ""));
+
+            String scraped = promMeterRegistry.scrape(TextFormat.CONTENT_TYPE_OPENMETRICS_100).replaceFirst("# EOF\r?\n?", "");
```
Comment (jgallimore) [inline implementation/src/main/java/io/smallrye/metrics/exporters/OpenMetricsExporter.java:39]: The value of the `scraped` variable is being ignored, so the output from this function is empty. I've corrected this in my local copy, and it appears to work ok.

Comment (jgallimore) [reply implementation/src/main/java/io/smallrye/metrics/exporters/OpenMetricsExporter.java:39]: Looks like this change was introduced here: https://github.com/smallrye/smallrye-metrics/commit/2e2d548744b525786e32a45f9328dee7eba24b33#diff-f7618f122be0f8b181e0c9f6a521fab33a07461c95fd5a2e1b03981d6b40641eR39

Comment (Channyboy) [reply implementation/src/main/java/io/smallrye/metrics/exporters/OpenMetricsExporter.java:38]: Ah, yes! sorry!  Forgot to re-rebase. pushed it now.

**Review by jgallimore — APPROVED** [review]
LGTM

**Pull request merged**